src/gc-tls.h

```diff
@@ -82,6 +82,7 @@ typedef struct {
     jl_gc_markqueue_t mark_queue;
     jl_gc_mark_cache_t gc_cache;
     _Atomic(size_t) gc_sweeps_requested;
+    _Atomic(uint8_t) gc_stack_sweep_requested;
```

Comment:
Perhaps use `size_t` here? Preventing the unlikely but painful debugging session from an overflow here seems worth the extra 7 bytes of memory in each TLS.